app/Http/Controllers/ApplicationController.php

```diff
@@ -58,6 +58,21 @@ public function show(JobApplication $application)
             $application->load(['veteran_status', 'citizenship_declaration', 'application_review', 'applicant.user']);
         }
 
+
+        // If the application status is draft then get data through the applicant model. Else, grab the data from the application itself.
+        if (!$application->isDraft()) {
```

Comment:
Oops, this check is backwards :P We want to go through the applicant if it IS a draft.

Reply:
Oops, forgot to switch that back :p I was testing it out.